.ci/release-update-github-page.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+chmod +x ./mvnw
```

Comment:
yes its kind of setup, but this way "highly coupled" as far away from its usage.